source/core_mqtt.c

```diff
@@ -796,11 +808,7 @@ static MQTTStatus_t discardPacket( const MQTTContext_t * pContext,
             elapsedTimeMs = calculateElapsedTime( getTimeStampMs(), entryTimeMs );
 
             /* Update remaining time and check for timeout. */
```

Comment:
This isn't accurate any more

Reply:
Removed the "Update remaining time" phrase